### ISSUE TITLE
Validation loss function fluctuates more compare to validation loss 

### ISSUE THREAD

**Issue by sneh-debug** [ellisdg/3DUnetCNN#262]
@ellisdg why validation loss fluctuations much compare to validation loss? I have used 219 images for training. Is the model overfitting?
